### DISCUSSION TITLE
Unkown device responding

### DISCUSSION THREAD

**Discussion by marcgagnon2112** [tasmota/tasmotizer#91] [General]
Hi everyone, 

Context : I'm using Tasmotizer on WIndows with a D1 Mini Pro attached to it.

Problem: I flashed few devices, so far, everything is good. BUT at one point, i figured out that the device responding was not the one I got in my hand! So there is another one, somewhere, which I cannot identify. The problem is that I would like to block it to be able to flash the good one.

I tried to find the wrong one to disable it. I went to the power panel and shut off all breakers one by one, but still, the device's config web page is showing. Maybe I get neighbour's one, don't have a clue.

Can I block somewhere the mac address of this device? Should I go instead through an ESPTool on the MiniPro to disable it from there? I'm clueless.

Merci from Montréal :)
Marc

**Comment by jziolkowski** [comment]
Tasmotizer works via serial, therefore any other device being present in the network has completely zero impact.

**Reply by marcgagnon2112** [reply]
I'm using it with a D1 Mini Pro (over the air)

**Reply by jziolkowski** [reply]
I still see no connection. Tasmotizer flashes over serial only.

**Reply by marcgagnon2112** [reply]
Windows -> Tasmo -> USB -> D1 Mini Pro -> Antenna -> over the air -> Device (i'm not connected with a wire on the device)

**Reply by jziolkowski** [reply]
Then tasmotizer is flashing your D1 Mini pro instead of the device

**Reply by marcgagnon2112** [reply]
Ho! I never thought about this possibility... How I can verify that?

**Reply by jziolkowski** [reply]
You don't have to. Tasmotizer _cannot_ flash the "device" OTA via D1 Mini

**Reply by jziolkowski** [reply]
How did you ever come up with such setup?

**Reply by marcgagnon2112** [reply]
well I think you right. I pressed the button to reset the Mini Pro and the web page didn't respond anymore. So its looks like I found the faulty device.

Funny! I used it, works fine for few devices and now, suddently, it is flashing itself... weird no? How I can prevent that?

**Reply by jziolkowski** [reply]
I think you have it wrong. There is no way that tasmotizer was able to flash another device OTA via something connected to USB

**Reply by sfromis** [reply]
It is not "flashing itself". Whenever you've got a compatible device connected over USB, Tasmotizer will flash it, without caring about what firmware may already be present. No "faulty device".

You can only prevent that by being careful. I make sure to only have devices connected via USB when I actually mean to either flash them, or use the serial port.

**Reply by marcgagnon2112** [reply]
Well, I don't know what to tell you. I did it. 
See @9:30 https://www.youtube.com/watch?v=dt5-iZc4_qU&ab_channel=digiblurDIY

**Reply by jziolkowski** [reply]
you are mistaking tuya-convert with tasmotizer

**Reply by sfromis** [reply]
Digiblur is not talking about Tasmotizer there, but Tuya-convert. An entirely different tool.

**Reply by marcgagnon2112** [reply]
I know, it is just to show the process. I got it worked on Windows too with Tasmotizer

**Reply by jziolkowski** [reply]
I know what Tasmotizer can do. It cannot flash OTA. You're free to think it does.
Either way, we've found your mysterious device. 
With that said, to quote Jenny Campbell: I'm out.

**Reply by marcgagnon2112** [reply]
Yes, thanks s lot. It is helping and "i'm not thinking that it is can be done", I did it. So you are free to think that it is not feasable. But still thanks for pointing the mysterious device for sure.

**Reply by vic42** [reply]
LOL, nice trolling attempt :1st_place_medal:
